src/style.css

```diff
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(/fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(/fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.bigSearchBar {
+  position: relative;
+  margin: 20px 0;
+}
+
+.bigSearchLine {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 22px;
+  color: #3d4e61;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.bigSearchLabel::before {
+  content: " ";
+  height: 19px;
+  width: 19px;
+  background-image: url(./images/Search.svg);
+  background-size: 19px 19px;
+  background-repeat: no-repeat;
+  background-position: center;
+  display: block;
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.bigSearchLine:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  outline: none;
+}
+
+.bigSearchLine:focus {
+  font-weight: 900;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  color: #3d4e61;
+  outline: none;
+}
+
+.smallSearchBar {
+  position: relative;
+}
+
+.smallSearchLine {
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  font-weight: 300;
+  line-height: 19px;
+  color: #3d4e61;
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.smallSearchLabel::before {
+  content: "";
+  height: 11px;
+  width: 11px;
+  background-image: url(./images/Search.svg);
+  background-size: 11px 11px;
+  background-repeat: no-repeat;
+  background-position: center;
+  display: block;
+  position: absolute;
+  top: 15px;
+  left: 13px;
+}
+
+.smallSearchLine:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  outline: none;
+}
+
+.smallSearchLine:focus {
+  font-weight: 900;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  color: #3d4e61;
+  outline: none;
```

Comment:
DRY - do not repeat yourself
part of the styles between big and small searches are the same
[checklist 2-nd](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md#:~:text=%5BSTYLES%5D%20%2D%20if%20you,fledged%20styled%20element.)